### ISSUE TITLE
The translation part of sl::CameraInformation::camera_imu_transform is incorrectly scaled

### ISSUE THREAD

**Issue by nvidiamfilimonov** [stereolabs/zed-examples#192]
Hello,

I've encountered the following issue while evaluating ZED-M camera with ZED SDK 2.8.3 on Ubuntu 18.04 LTS:
The translation part of sl::CameraInformation::camera_imu_transform is incorrectly scaled.

With
```
    sl::InitParameters initParameters;
    initParameters.coordinate_units = sl::UNIT_MILLIMETER;
```

camera_To_imu_translation is equal to
>x: 1.50000e-02 y: 2.00000e-03 z: -1.00000e-03

The translation of 1.5* 10^-2 mm is obviously incorrect.

With `initParameters.coordinate_units = sl::UNIT_METER`;
camera_To_imu_translation is equal to
>x: 1.50000e-05 y: 2.00000e-06 z: -1.00000e-06

Below is the C++ code example demonstrating the issue:
```
#include <chrono>
#include <iostream>
#include <thread>

#include <sl/Camera.hpp>


void check(bool ok)
{
    if (!ok)
    {
        std::cerr << "Error\n";
        std::exit(1);
    }
}

void printSDKVersion()
{
    int major;
    int minor;
    int patch;

    getZEDSDKBuildVersion(major, minor, patch);

    std::cout << "SDK version " << major << '.' << minor << '.' << patch << '\n';
}

void printTranslation(const char* label, const sl::Translation& translation)
{
    std::cout << label << '\n' << std::scientific << std::setprecision(5);
    std::cout << "x: " << translation.x << " y: " << translation.y << " z: " << translation.z;
    std::cout << "\n";
}

void printTransform(const char* label, const sl::Transform& transform)
{
    sl::Transform copy{transform};

    std::cout << label << '\n' << std::scientific << std::setprecision(5);
    for (int u = 0; u < 4; ++u)
    {
        std::cout << "   ";
        for (int v = 0; v < 4; ++v)
        {
            std::cout << ' ' << copy(u, v);
        }
        std::cout << '\n';
    }
}

void printCameraImuTransform(const sl::Transform& transform)
{
    //printTransform("camera_T_imu_transform", transform);
    printTranslation("camera_T_imu_translation", transform.getTranslation());
}

void printCameraInformation(sl::Camera& camera)
{
    std::cout << "cameraInformation\n";
    const sl::CameraInformation cameraInformation{camera.getCameraInformation()};
    std::cout << "SN " << cameraInformation.serial_number << " FW version " << cameraInformation.firmware_version << "\n";
    printCameraImuTransform(cameraInformation.camera_imu_transform);
}

int main(int argc, char **argv)
{

    // Create a ZED camera object
    sl::Camera camera;

    // Set configuration parameters
    sl::InitParameters initParameters;
    initParameters.coordinate_units = sl::UNIT_METER;//sl::UNIT_MILLIMETER;
    initParameters.camera_resolution = sl::RESOLUTION_VGA;
    initParameters.camera_fps = 60;

    // Open the camera
    sl::ERROR_CODE err = camera.open(initParameters);
    check(err == sl::SUCCESS);

    printSDKVersion();
    printCameraInformation(camera);
    std::cout << '\n';

    camera.close();

    return 0;
}
```

**Comment by obraun-sl** [comment]
Thanks for reporting this issue. There is indeed a wrong 0.001f scale applied in the sdk. 
This will be fixed in next patch. 
In the meantime, the workaround is to multiplied by 1000 whatever the units choosen.

**Comment by adujardin** [comment]
This issue was fixed from ZED SDK [2.8.4](https://www.stereolabs.com/developers/release/2.8/)
